Added the MinnowBoard Turbot, replaced broken links.

Mark-van-der-Pol opened the pull request:
MinnowMax is superseded, replaced with the Minnow Turbot - 
updated the web links to point to the legacy board pages, and the new board where appropriate.
Pin 26 - Added a pin function for the I2S MCLK

There is more work to do to capture and document all the changes.

Signed-off-by: |\\/|ark van der Pol <markx.van.der.pol@intel.com>

arfoll commented:
Thanks a lot for the commit, it looks good. However as per our contributors guide please sign off your commit (see CONTRIBUTING.md). Thanks!

Also do you have any links to the mmax obsoletion by the turbot? Obviously it's the newer/more reliable board but I've never seen the "obsoletion" language used previously?

Mark-van-der-Pol commented:
You are right – I made up it is obsolete – there is no language I can find that states that, though I would swear I did read it somewhere. I will amend it to say supersedes.

And sign off on the patch per contribution guide.

|\/|ark.

From: Brendan Le Foll [mailto:notifications@github.com]
Sent: Friday, April 14, 2017 1:51 AM
To: intel-iot-devkit/mraa <mraa@noreply.github.com>
Cc: Van Der Pol, MarkX <markx.van.der.pol@intel.com>; Author <author@noreply.github.com>
Subject: Re: [intel-iot-devkit/mraa] Added the Minnow Turbot, replaced broken links. (#706)


Thanks a lot for the commit, it looks good. However as per our contributors guide please sign off your commit (see CONTRIBUTING.md). Thanks!

Also do you have any links to the mmax obsoletion by the turbot? Obviously it's the newer/more reliable board but I've never seen the "obsoletion" language used previously?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/intel-iot-devkit/mraa/pull/706#issuecomment-294118863>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AZPcfSzANWNk4WAE4Ng5JWmEzYMTdWk_ks5rvzNxgaJpZM4M9bRQ>.

arfoll commented:
Any updates on the signoff? would love to merge this!

No worries on the deprecation, I saw that and figured I don't know the politics between the max & turbot projects or if they're at all different? but I don't want to give an impression that the mraa project takes any position on what's considered deprecated :)

whbruce commented:
Max and Turbot are very similar designs from different manufacturers.  The Max is no longer being manufactured. The Turbot has some design changes to get FCC approval and can support the quad core sku of Baytrail. That said, there are plenty of Max variants out there so we shouldn't remove support from mraa.

Mark-van-der-Pol commented:
I updated my original commit comment with the sign-off. Is that not visible? Or is there something else I should do?
|\/|ark.

From: Brendan Le Foll [mailto:notifications@github.com]
Sent: Wednesday, April 19, 2017 9:50 AM
To: intel-iot-devkit/mraa <mraa@noreply.github.com>
Cc: Van Der Pol, MarkX <markx.van.der.pol@intel.com>; Author <author@noreply.github.com>
Subject: Re: [intel-iot-devkit/mraa] Added the Minnow Turbot, replaced broken links. (#706)


Any updates on the signoff? would love to merge this!

No worries on the deprecation, I saw that and figured I don't know the politics between the max & turbot projects or if they're at all different? but I don't want to give an impression that the mraa project takes any position on what's considered deprecated :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/intel-iot-devkit/mraa/pull/706#issuecomment-295341210>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AZPcfSpEe11KDIP8mv98jqPfdR9mQLyLks5rxjsxgaJpZM4M9bRQ>.

arfoll commented:
you updated the commit comment on github, that side has no real effect on the commit itself.

@whbruce the max is out of prod? That's fair to call it deprecated then. I was worried both boards where still being made so calling one deprecated could annoy one manufacturer :)

Mark-van-der-Pol commented:
I didn’t mean to suggest MinnowBoard MAX support is removed – it is that the MinnowBoard Turbot is what is now available from vendors, and it is backward compatible so transparently works as a MinnowBoard MAX with the libraries. There is nothing to do in the code.

And the description file was using outdated and broken links which I updated.
|\/|ark.

From: Henry Bruce [mailto:notifications@github.com]
Sent: Wednesday, April 19, 2017 9:58 AM
To: intel-iot-devkit/mraa <mraa@noreply.github.com>
Cc: Van Der Pol, MarkX <markx.van.der.pol@intel.com>; Author <author@noreply.github.com>
Subject: Re: [intel-iot-devkit/mraa] Added the Minnow Turbot, replaced broken links. (#706)


Max and Turbot are very similar designs from different manufacturers. The Max is no longer being manufactured. The Turbot has some design changes to get FCC approval and can support the quad core sku of Baytrail. That said, there are plenty of Max variants out there so we shouldn't remove support from mraa.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/intel-iot-devkit/mraa/pull/706#issuecomment-295344375>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AZPcfT2CFVoy-rfrbHcapFc1mr13jSEPks5rxj0TgaJpZM4M9bRQ>.

Propanu commented:
Might want to fix spellings for board and assignment on lines 11 and 62 while we're at it.

alext-mkrs commented on docs/minnow_max.md:
```diff
@@ -1,12 +1,14 @@
+=============================================
 MinnowBoard MAX is an open hardware embedded board designed with the Intel(R)
-Atom(TM) E38xx series SOC (Fromerly Bay Trail).
+Atom(TM) E38xx series SOC (Fromerly Bay Trail). The MinnowBoard Turbot is a 
```
typo + style (no need for capital F): Fromerly

arfoll commented:
Fixed up and merged. Thanks alot for the contribution!
